src/main/java/core/basesyntax/DataConverterImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the empty line at the start of the class definition.